demo/actions.py

```diff
@@ -724,15 +735,23 @@ def run(self, dispatcher, tracker, domain):
         # If we're in a TwoStageFallback we need to look back two more user utterance to get the actual text
         if search_text == "/technical_question{}" or search_text == "/deny":
             last_user_event = get_last_event_for(tracker, "user", skip=3)
-            search_text = last_user_event.get("text")
+            if last_user_event:
+                search_text = last_user_event.get("text")
+            else:
+                dispatcher.utter_message(text="Sorry, I can't answer your question.")
+                return []
 
         # Search forum
         discourse = DiscourseAPI("https://forum.rasa.com/search")
         doc_list = discourse.query(search_text)
         doc_list = doc_list.json()
 
-        forum = discourse.get_discourse_links(doc_list.get("topics"), 0)
-        forum += "\n" + discourse.get_discourse_links(doc_list.get("topics"), 1)
+        if doc_list:
```

Comment:
if not doc_list (also can we please rename this bc i thought this was the docs search lol), can we then also say sth like "sorry, i don't know how to answer that yet" ?

Reply:
I renamed `doc_list` and also now give a message with recommendation that they post their question on the forum.